### PR TITLE
refactor(editor): Fix TypeScript issues in template code (no-changelog)

### PR DESCRIPTION
**Pull request by elsmr**
## Review / Merge checklist
- [ ] PR title and summary are descriptive. **Remember, the title automatically goes into the changelog. Use `(no-changelog)` otherwise.** ([conventions](https://github.com/n8n-io/n8n/blob/master/.github/pull_request_title_conventions.md))
- [ ] [Docs updated](https://github.com/n8n-io/n8n-docs) or follow-up ticket created.
- [ ] Tests included.
   > A bug is not considered fixed, unless a test is added to prevent it from happening again.
   > A feature is not complete without tests.

### CODE REVIEW

File: packages/editor-ui/src/views/TemplatesCollectionView.vue [hunk 1]
```diff
@@ -21,18 +21,21 @@
...
 				<div :class="$style.mainContent">
-					<div v-if="loading || (collection && collection.description)" :class="$style.markdown">
+					<div
+						v-if="loading || (collection && 'description' in collection)"
+						:class="$style.markdown"
+					>
 						<n8n-markdown
-							:content="collection && collection.description"
-							:images="collection && collection.image"
+							:content="collection && 'description' in collection && collection.description"
```
Comment (mutdmour) [inline packages/editor-ui/src/views/TemplatesCollectionView.vue:29]: nitpick - maybe`isFullTemplatesCollection` typeguard here might be clearer. Same for other places here where you use this pattern.

**Review by mutdmour — DISMISSED** [review]
LGTM. Would be good to test the templates feature separately since it's hard to test now. Not sure if e2e run against it still

**Comment by elsmr** [comment]
Was able to test it with Milorad's help, everything OK!

**Review by mutdmour — APPROVED** [review]

**Comment by janober** [comment]
Got released with [`n8n@1.45.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.45.0)

**Pull request merged**